### PR TITLE
Add Term: HTML Tags `<select>`,`<option>`

### PR DESCRIPTION
**Pull request by SSwiniarski**
### Description

Add terms for HTML Tags `<select>` and `<option>`, from the Target Keyword list

### Type of Change

- Adding a new entry

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by lukeecart — APPROVED** [review]
Looks good to me. I see some of your commits say "yarn format". Are the git hooks (husky) not running?

File: content/html/concepts/tags/terms/select/select.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+```html
+<select name="dropdown name">
+  <option value="value of option 1">Text for option 1</option>
+  <option value="value of option 2">Text for option 2</option>
+  <option value="value of option 3">Text for option 3</option>
+</select>
+```
+
+The `<select>` element is given a `name` attribute to reference it in the form data, and the `<option>` elements will each have a `value` attribute to pass along when the form is submitted with that option selected.
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/select/select.md:29]:
```suggestion
The `<select>` element is referenced in the form data with a `name` attribute. Inside, the `<option>` elements will each have a `value` attribute that is submitted with the form if that option is selected.
```

File: content/html/concepts/tags/terms/select/select.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+  - 'Tags'
+  - 'UI'
+CatalogContent:
+  - 'learn-html'
+  - 'paths/front-end-engineer-career-path'
+---
+
+The `<select>` tag creates a dropdown list where the user can select from among a number of [`<option>`](https://www.codecademy.com/resources/docs/html/tags/option) elements.
+
+The `<select>` element is typically used with the `<option>` elements to gather user input in a [form](https://www.codecademy.com/resources/docs/html/forms).
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/select/select.md:17]:
```suggestion
The `<select>` element is normally used with `<option>` elements for gathering user input in a [form](https://www.codecademy.com/resources/docs/html/forms).
```

File: content/html/concepts/tags/terms/option/option.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+  <option value="value of option 2">Text for option 2</option>
+  <option value="value of option 3">Text for option 3</option>
+</select>
+```
+
+The `<select>` element is given a `name` attribute to reference it in the form data, and the `<option>` elements will each have a `value` attribute to pass along when the form is submitted with that option selected.
+
+## Example
+
+Here's a simple form that allows the user to select one of four different types of meals.
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/option/option.md:33]:
```suggestion
The example form below allows the user to select one of four different types of meals:
```

**Review by Dusch4593 — CHANGES_REQUESTED** [review]
@SSwiniarski Left a few comments/suggestions for you. 😄

**Comment by SSwiniarski** [comment]
@Dusch4593, I implemented your suggestions.

File: content/html/concepts/tags/terms/select/select.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+<html>
+  <head> </head>
+  <body>
+    <form>
+      <label for="food">Select Meal Type</label>
+      <select name="food" id="food">
+        <option value"1">Beef</option>
+        <option value"2">Chicken</option>
+        <option value"3">Fish</option>
+        <option value"4">Vegetarian</option>
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/select/select.md:45]:
```suggestion
        <option value="1">Beef</option>
        <option value="2">Chicken</option>
        <option value="3">Fish</option>
        <option value="4">Vegetarian</option>
```

File: content/html/concepts/tags/terms/option/option.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+<html>
+  <head> </head>
+  <body>
+    <form>
+      <label for="food">Select Meal Type</label>
+      <select name="food" id="food">
+        <option value"1">Beef</option>
+        <option value"2">Chicken</option>
+        <option value"3">Fish</option>
+        <option value"4">Vegetarian</option>
```
Comment (Dusch4593) [inline content/html/concepts/tags/terms/option/option.md:45]:
```suggestion
        <option value="1">Beef</option>
        <option value="2">Chicken</option>
        <option value="3">Fish</option>
        <option value="4">Vegetarian</option>
```

**Review by Dusch4593 — APPROVED** [review]
Just 2 more suggested edits from me but we should be good for the next review 👍🏻

**Review by KTom101 — CHANGES_REQUESTED** [review]
@SSwiniarski I don't have much to suggest for this. I only suggested adding attributes sections for the other attributes used with `<option>` and `<select>`.

**Comment by SSwiniarski** [comment]
@KTom101, I implemented your suggestions

**Review by KTom101 — APPROVED** [review]
@SSwiniarski Thank you! This looks good to me.

@Dusch4593 Anything else you want to add to this?

**Review by Dusch4593 — APPROVED** [review]
LGTM! 👍🏻

**Comment by Dusch4593** [comment]
Live entries here:

- `<option>`: https://www.codecademy.com/resources/docs/html/tags/option
- `<select>`: https://www.codecademy.com/resources/docs/html/tags/select

**Pull request merged**